src/components/Checkbox/src/Checkbox.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <el-checkbox
+    :true-label="label"
+    :false-label="n_lable"
+    :disabled="disabled"
+    v-model="current"
+  >
+    {{ display }}
+  </el-checkbox>
+</template>
+
+<script>
+export default {
+  name: 'Checkbox',
+
+  props: {
+    value: {
+      type: [String, Number, Boolean],
+      required: true
+    },
+    label: {
+      type: [String, Number, Boolean],
+      required: true
+    },
+    n_lable: {
+      type: [String, Number, Boolean],
+      required: true
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    },
+    display: {
+      type: [String, Number],
+      required: true
+    }
+  },
+
+  computed: {
+    current: {
+      get: function() {
+        return this.value
```

Comment:
The "value" prop is not get/set anywhere. This should return the value of the checkbox.